AmericanMedicalAssociation/ama-style-guide-2
# EWL-6547 Updates resource page schedule table to stack and use the mobile view

## JIRA Ticket(s)
- [EWL-6547: Resource page - mobile. tables not showing complete data](https://issues.ama-assn.org/browse/EWL-6547)


## Description:

- Adds wrappers need to target the resource page schedule table
- Adds basicTables.js library which enables the the mobile and desktop views (For more info visit: http://www.jerrylow.com/basictable/)
- Adds classes to table headers
- Adds class names to tables with index


## To Test:
- for testing instructions visit https://github.com/AmericanMedicalAssociation/ama-d8/pull/1060
- also visit http://localhost:3000/?p=pages-resource
- click on the schedule tab
- observe the schedules table has stacked events with grey and white zebra striping
- observe the schedule matches the zeplin mockup https://app.zeplin.io/project/59dd1f9c36a8b21d569bb9e8/screen/5ac3997560cdf96b3b16ea03
- click on the table tab
- observe it has default table functionality
- resize your browser to mobile size
- observe the table is stacked 


## Visual Regressions
A report is available [here](http://htmlpreview.github.io/?https://github.com/AmericanMedicalAssociation/ama-style-guide-2/blob/visual-regression-testing-artifact/bugfix/EWL-6262-subcat-stacking-bug/html_report/index.html).

## Relevant Screenshots/GIFs
![cpt _editorial_panel_meeting_process_calendar___american_medical_association](https://user-images.githubusercontent.com/2271747/48501888-e2a7e500-e803-11e8-89b5-b4b600c061e8.png)


## Remaining Tasks
This is has a dependency in D8 https://github.com/AmericanMedicalAssociation/ama-d8/pull/1060


## Additional Notes
The table info on the resource page needs to be moved to the schedule tab in D8

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
